lib/linguist/languages.yml

```diff
@@ -4343,6 +4343,17 @@ STON:
   tm_scope: source.smalltalk
   ace_mode: text
   language_id: 336
+Svelte:
+  type: markup
+  tm_scope: text.html.svelte
+  group: HTML
+  ace_mode: html
+  codemirror_mode: htmlmixed
+  codemirror_mime_type: text/html
+  color: "#e34c26"
+  extensions:
+  - ".svelte"
```

Comment:
I counted 117 repositories by 102 users from the search query you provided. That a bit low considering our usual requirement of *hundreds* of repositories, but it's unlikely we'll get any conflicts in the future given the extension. @lildude What do you think?

Reply:
Unfortunately, that usage is not high enough for inclusion right now, it's barely over a _hundred_ let alone _hundred**s**_.